sdk/tables/Azure.Data.Tables/src/TableRestClient.cs

```diff
@@ -77,6 +77,7 @@ public async Task<Response<List<Response>>> SendBatchRequestAsync(HttpMessage me
                         if (responses.Length == 1 && responses.Any(r => r.Status >= 400))
                         {
                             var ex = await _clientDiagnostics.CreateRequestFailedExceptionAsync(responses[0]).ConfigureAwait(false);
```

Comment:
My only question is whether we expect callers to know this is a partial-success scenario.   If they resend the same batch, does it potentially cause duplication or is everything safely idempotent?

Reply:
Attempting to resend the same batch will throw:

https://github.com/Azure/azure-sdk-for-net/blob/2c6f765c16d9d63b5b6e59b0404129c192464206/sdk/tables/Azure.Data.Tables/src/TableTransactionalBatch.cs#L229-L231

Reply:
But you are correct that there isn't a great story right now for failure recovery. Currently, the only option is to totally recreate the batch and try again. It's something we've discussed as a feature team to improve at some point.